cypress/integration/examples/navigation.spec.js

```diff
@@ -0,0 +1,56 @@
+/// <reference types="cypress" />
+
+context('Navigation', () => {
```

Comment:
The test suite is not related to the task description. Update the test suite to cover the Book Store app functionality (login, add book, and delete book) using Cypress.